### PR TITLE
ES2020 version

### PR DESCRIPTION
**Pull request by Dr-Horv**

### CODE REVIEW

**Comment by dubbe** [comment]
I tried to add a way to easily change the bot settings in bot.js.

File: src/utils.js [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+
+  getCharacter(position) {
+    const playerId = this.getPlayerIdAtPosition(position);
+    const characterInfo = this.characterInfoMap[playerId];
+    return createCharacterTile(characterInfo);
+  }
+
+  getPlayerIdAtPosition(position) {
+    for (const characterInfo of this.map.characterInfos) {
+      if (characterInfo.position == position) {
```
[the remark this answers is not in the document]
Comment (Dr-Horv) [reply src/utils.js:333]: :scream: I am immensely shamed

**Comment by Dr-Horv** [comment]
@maxdavidson updated, thanks for the feedback!

File: src/utils.js [hunk 2]
```diff
@@ -99,54 +107,68 @@ export class Coordinate {
...
-    for (const ci of map.characterInfos) {
-      this.characterInfoMap.set(ci.id, ci);
-      this.positionToTiles.set(ci.position, createCharacterTile(ci));
+    for (const characterInfo of map.characterInfos) {
+      this.#characters.set(characterInfo.id, characterInfo);
+      this.#tiles.set(characterInfo.position, createCharacterTile(characterInfo.id));
+
+      const colourTile = createColourTile(characterInfo.id);
+      for (const colouredPosition of characterInfo.colouredPositions) {
+        this.#tiles.set(colouredPosition, colourTile);
```
Comment (Dr-Horv) [inline src/utils.js:170]: You realize that a bot can be standing on a coloured tile, I.e. the position blue is standing on will be coloured blue, so no you're overwritting the character tile with the colour tile

**Pull request merged**